projects/storefrontlib/src/shared/components/anonymous-consents-dialog/anonymous-consent-dialog.component.html

```diff
@@ -0,0 +1,77 @@
+<div
+  class="modal-dialog modal-dialog-centered modal-lg"
```

Comment:
imho we should not have _any_ notion of a modal inside this component. The modal is our opiniated choice, which should be solely driven by CSS.

Can we change this to:
- drop the specific classes
- work with the component selector class `cx-anonymous-consent-dialog` and extend that class from modal class(es)

Reply:
Agreed